apps/taxes/src/modules/taxes/distribute-discount.ts

```diff
@@ -0,0 +1,24 @@
+import { numbers } from "./numbers";
+
+// ? shouldn't it be used in all providers?
+
+export function distributeDiscount(discountSum: number, prices: number[]) {
```

Comment:
Might be out of scope of this (if so, please ignore), but once had edge case for one of my clients - how the discount should be calculated if order contains returned product?

How to reproduce: 
- create order with a few order lines
- apply discount on all the lines (it will be distributed)
- refund one of the order lines
- calculate taxes again

Reply:
You are breaking my brain man, I will add a comment with this scenario and look into it later.

Reply:
@krzysztofwolski I think after Order is placed, taxes are not calculated anymore